compiler/src/dmd/expressionsem.d

```diff
@@ -875,6 +875,13 @@ Lagain:
                 if (sc.setUnsafePreview(global.params.systemVariables, false, loc,
                     "cannot access `@system` variable `%s` in @safe code", sd))
                 {
+                    if (auto v = sd.isVarDeclaration())
+                    {
+                        if (v.systemInferred)
+                            errorSupplemental(v.loc, "`%s` is inferred to be `@system` from its initializer", v.toChars());
+                        else
+                            errorSupplemental(v.loc, "`%s` is located here", v.toChars());
```

Comment:
how about _declared_ instead of located ?

Reply:
Done